pkg/certs/certgen.go

```diff
@@ -255,4 +255,4 @@ func encodeToPEMKey(key *rsa.PrivateKey) []byte {
 
 func encodeToPEMPublicKey(key []byte) []byte {
 	return pem.EncodeToMemory(&pem.Block{Type: "PUBLIC KEY", Bytes: key})
-}
+}
```

Comment:
FYI, no need to delete the terminal linebreak; that deletion introduces alarming red stuff in the view through GitHub.